### PR TITLE
various fixes for mkcloud

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/jenkins/ci.suse.de/openstack-mediacheck.xml [hunk 1]
```diff
@@ -62,14 +61,20 @@
...
       </values>
     </hudson.matrix.LabelAxis>
   </axes>
-  <combinationFilter>(["Devel:Cloud:3", "Devel:Cloud:4", "Devel:Cloud:5", "Devel:Cloud:5/SLE_12", "Devel:Cloud:6", "Devel:Cloud:6/SLE_12"].contains(project) || subproject == ":")</combinationFilter>
+  <combinationFilter>(["Devel:Cloud:4", "Devel:Cloud:5", "Devel:Cloud:5/SLE_12", "Devel:Cloud:6", "Devel:Cloud:6/SLE_12"].contains(project) || subproject == ":")</combinationFilter>
   <builders>
     <hudson.tasks.Shell>
       <command># temporary, similar to update_automation (not that advanced)
+if [ ! -e  /root/github.com/SUSE/cloud/ ] ; then
+  mkdir -p /root/github.com/SUSE
```
Comment (tboerger) [inline scripts/jenkins/ci.suse.de/openstack-mediacheck.xml]: Why not simply ```git clone git@github.com:SUSE/cloud.git /root/github.com/SUSE/cloud```?

Comment (bmwiedemann) [inline scripts/jenkins/ci.suse.de/openstack-mediacheck.xml]: so something like
```bash
d=/root/github.com/SUSE/cloud/
[[ -e $d ]] || git clone git@github.com:SUSE/cloud.git $d
```

Comment (dirkmueller) [inline scripts/jenkins/ci.suse.de/openstack-mediacheck.xml]: I did not write that code, I'm just committing the template that was changed on the server by someone..

**Comment by dirkmueller** [comment]
@bmwiedemann how can you submit a fix to GM4 ? its frozen..

**Pull request merged**